pkg/api/server/server.go

```diff
@@ -48,24 +48,47 @@ import (
 )
 
 type APIServer struct {
-	http.Server        // Where the HTTP work happens
+	http.Server        // The  HTTP work happens here
 	*schema.Decoder    // Decoder for Query parameters to structs
-	context.Context    // Context for graceful server shutdown
+	context.Context    // Context to carry objects to handlers
 	*libpod.Runtime    // Where the real work happens
 	net.Listener       // mux for routing HTTP API calls to libpod routines
 	context.CancelFunc // Stop APIServer
 	*time.Timer        // Hold timer for sliding window
 	time.Duration      // Duration of client access sliding window
 }
 
-// NewServer will create and configure a new API HTTP server
+// Number of seconds to wait for next request, if exceeded shutdown server
+const (
+	DefaultServiceDuration   = 300 * time.Second
```

Comment:
any thoughts about adding this to the conf file?  5 minutes seems very reasonable though in probably 99.999999999999% of the cases.

Reply:
it is the default, but is overriden by user with an option.